### DISCUSSION TITLE
请问在本地怎么start

### DISCUSSION THREAD

**Discussion by kevin-hans** [alibaba/designable#351] [Q&A]
![image](https://user-images.githubusercontent.com/21983429/211199946-137035e9-2056-4331-b5d9-47a6d009ce1b.png)
![image](https://user-images.githubusercontent.com/21983429/211199961-99e7ffb8-735c-4ac8-873f-87acdd9ac970.png)

我是
git clone
cd 
npm install
或者 yarn add

**Comment by hfcel** [comment]
https://www.yuque.com/xjchenhao/development/rncluc
参见这篇
